### PR TITLE
Introduce new kafka version related status fields

### PR DESCRIPTION
**Pull request by baluchicken**
| Q               | A
| --------------- | ---
| Bug fix?        | no|
| New feature?    |yes|
| API breaks?     | no|
| Deprecations?   | no|
| License         | Apache 2.0

### What's in this PR?

This PR introduces new kafka version related status fields to the CRD.

### CODE REVIEW

File: pkg/sdk/v1beta1/common_types.go [hunk 1]
```diff
@@ -151,6 +157,8 @@ type BrokerState struct {
 	PerBrokerConfigurationState PerBrokerConfigurationState `json:"perBrokerConfigurationState"`
 	// ExternalListenerConfigNames holds info about what listener config is in use with the broker
 	ExternalListenerConfigNames ExternalListenerConfigNames `json:"externalListenerConfigNames,omitempty"`
+	// KafkaVersion holds the kafka version with the used docker image version
```
Comment (stoader) [inline pkg/sdk/v1beta1/common_types.go:160]:
```suggestion
	// KafkaVersion holds the kafka version of the broker in semver format
```

File: pkg/sdk/v1beta1/common_types.go [hunk 2]
```diff
@@ -44,6 +44,12 @@ type PerBrokerConfigurationState string
 // ExternalListenerConfigNames type describes a collection of external listener names
 type ExternalListenerConfigNames []string
 
+// KafkaVersion type describes the kafka version and docker version
+type KafkaVersion struct {
+	Version        string `json:"version"`
```
Comment (chrisgacsal) [inline pkg/sdk/v1beta1/common_types.go:49]: Indentation alert. 🚨

Comment (baluchicken) [reply pkg/sdk/v1beta1/common_types.go:49]: Thanks

File: pkg/sdk/v1beta1/common_types.go [hunk 3]
```diff
@@ -46,8 +46,8 @@ type ExternalListenerConfigNames []string
 
 // KafkaVersion type describes the kafka version and docker version
 type KafkaVersion struct {
-	Version        string `json:"version"`
-	DockerImage string `json:"dockerImage"`
+	Version string `json:"version"`
```
Comment (stoader) [inline pkg/sdk/v1beta1/common_types.go]:
```suggestion
    // Version holds the current version of the broker in semver format
	Version string `json:"version"`
```

File: pkg/sdk/v1beta1/common_types.go [hunk 4]
```diff
@@ -46,8 +46,8 @@ type ExternalListenerConfigNames []string
...
+	Image   string `json:"image"`
```
Comment (stoader) [inline pkg/sdk/v1beta1/common_types.go]:
```suggestion
    // Image specifies the current docker image of the broker 
	Image   string `json:"image"`
```

**Review by stoader — CHANGES_REQUESTED** [review]

**Review by stoader — APPROVED** [review]

**Review by schultheiszn — APPROVED** [review]

**Review by adamantal — APPROVED** [review]

**Pull request merged**